notebooks/244-depth-estimation-videpth/244-depth-estimation-videpth.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    !pip install lightning timm==0.6.12
</pre><p>please use -q parameter for reduce pip logs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>